homeassistant/components/vallox/config_flow.py

```diff
@@ -30,6 +30,8 @@
     ValloxApiException,
 )
 
+INTEGRATION_DOCS_URL = "https://www.home-assistant.io/integrations/vallox"
```

Comment:
You could call async_get_integration and get the documentation url for there so we don't have to maintain it in two places

Reply:
Even better, TIL 👍 Fixed.